pkgs/tools/misc/ntfy-sh/default.nix

```
@@ -15,8 +21,21 @@ buildGoModule rec {
 
   doCheck = false;
 
+  nativeBuildInputs = with pkgs; [
+    debianutils
+    mkdocs
+    nodejs
+    python3
+    python3Packages.mkdocs-material
+    python3Packages.mkdocs-minify
+  ];
+
   preBuild = ''
-    make cli-deps-static-sites
+    sed -i 's# /bin/echo# echo#' Makefile
+    sed -i 's#react-scripts build#DISABLE_ESLINT_PLUGIN=true react-scripts build#' web/package.json
+    ln -s ${nodeDependencies}/lib/node_modules web/node_modules
+    make web-build
+    make docs-build
```

I think these can be collapsed?

```suggestion
    make web-build docs-build
```